docs/content/docs/test/ScreenChecks.md

```diff
@@ -0,0 +1,96 @@
+# Screen checks
```

Comment:
What about https://avito-tech.github.io/avito-android/docs/test_framework/testframework/#screenchecks ?
Will you replace that text by the link?

Reply:
I didn't know about the existance of that page. Will merge them